CMakeLists.txt

```diff
@@ -114,34 +114,34 @@ install(DIRECTORY ${CMAKE_BINARY_DIR}/mod/
 
 install(EXPORT mocsyTargets
   FILE mocsyTargets.cmake
```

Comment:
Can we please change to `mocsy-targets.cmake` for consistency with other scripts (see below)